### PR TITLE
add qualimap/rnaseq nf-test test

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Comment by Emiller88** [comment]
As we go through these with rnaseq and sarek and update the "maintainers" meta fields to have the team names? Or just the CODEOWNERS(or both?)

**Comment by maxulysse** [comment]
> As we go through these with rnaseq and sarek and should we update the "maintainers" meta fields to have the team names? Or just the CODEOWNERS(or both?)

can we have a team as a codeowner?

**Review by SPPearce — COMMENTED** [review]
Looks fine to me, but I can't vouch that all the tags etc are correct - do we have a checklist for that somewhere?
Seems a bit strange that the original test was generating the bam file via STAR rather than just using one from test-data?

**Comment by maxulysse** [comment]
> Looks fine to me, but I can't vouch that all the tags etc are correct - do we have a checklist for that somewhere?
> Seems a bit strange that the original test was generating the bam file via STAR rather than just using one from test-data?

Yeah, found it was easier just to use an existing bam to be honest

File: modules/nf-core/qualimap/rnaseq/tests/main.nf.test [hunk 1]
```diff
@@ -0,0 +1,38 @@
+nextflow_process {
+
+    name "Test Process QUALIMAP_RNASEQ"
+    script "../main.nf"
+    process "QUALIMAP_RNASEQ"
+    tag "modules"
+    tag "modules_nfcore"
+    tag "qualimap/rnaseq"
```
Comment (sateeshperi) [inline modules/nf-core/qualimap/rnaseq/tests/main.nf.test:8]:
```suggestion
    tag "qualimap"
    tag "qualimap/rnaseq"
```

File: modules/nf-core/qualimap/rnaseq/tests/main.nf.test [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    test("BAM") {
```
Comment (sateeshperi) [inline modules/nf-core/qualimap/rnaseq/tests/main.nf.test:10]:
```suggestion
    test("homo_sapiens [BAM]") {
```

**Review by sateeshperi — COMMENTED** [review]
lets not remove the pytests just now

**Review by sateeshperi — APPROVED** [review]

**Pull request merged**